_posts/2021-06-22-choosing-right-transformation.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Choosing the Right Transformation
+author:
+  display_name: Jake Adams
+  email: jdadams@utah.gov
+date: 2021-06-21 8:30:00
+categories:
+  - Featured
+  - SGID Blog
+tags:
+  - sgid
+  - spatial analysis
+  - data
+---
+
+{% include embedded_video.html embed_url="https://player.vimeo.com/video/565826461" %}
+_From the I Hate Slivers department_
+{: .flex .flex--center}
+
+Modern GIS software goes to great lengths to automate the mundane tasks and hide the boring stuff from us. As a relative newcomer to the field, I've heard the old-timers talk about how spoiled we are with on-the-fly projections (now excuse me while I get off their lawn). But really, we are. Being able to display and process data in a variety of projections without first having to reproject it all saves both time and sanity.
+
+This is even more true with the rise of web gis and external databases. If I were working for a county, I could pull in local data in a State Plane projection, OpenSGID layers in UTM 12N NAD83, and AGOL layers in Web Mercator and they'd all look just about right.
+
+"Just about."
+
+[![Close but no cigar]({% link images/transformations_measurement.jpg %}){:width="800px"}]({% link images/transformations_measurement.jpg %})
```

Comment:
**suggestion**: this width won't look nice on mobile but /shrug

Reply:
Do you have a recommendation for handling the size better? I did this for all the screenshots that were over 800px wide to keep it from being too big on desktop, but I can totally see it being a problem for mobile. Is there a way to say "browser width up to a max of 800"?

Reply:
you could try using percentages. there are min-width and max-width css properties